tests/unittests/cli/test_fast_api.py

```diff
@@ -94,6 +94,19 @@ def _event_3():
   )
 
 
+def _event_state_delta(state_delta: dict[str, Any]):
+  return Event(
+      author="dummy agent",
+      invocation_id="invocation_id",
+      content=types.Content(
+          role="model",
+          parts=[
+              types.Part(text=json.dumps(state_delta, sort_keys=True))
+          ],
+      ),
```

Comment:
An event with state_delta does not have a content, but has event_actions.state_delta set. https://github.com/google/adk-python/blob/main/src/google/adk/events/event_actions.py#L45